pkg/api/v1alpha1/README.md

```diff
@@ -0,0 +1,24 @@
+# Create new webhooks
+
+## Install kubebuilder
+```sh
+make hack/tools/bin/kubebuilder
+```
+
+## Add API to PROJECT https://github.com/aws/eks-anywhere/blob/main/PROJECT
+```sh
+run ./hack/kubebuilder.sh create api --group anywhere --version v1alpha1 --kind WhateverKind
+``` 
+
+## Create new webhooks with kubebuilder
+Since we use a non standard (according to kubebuilder) repo structure, kubebuilder commands won't work. For this purpose, we have a script that temporally changes our folder structure to one that kubebuilder understands and restores the original one after executing the kubebuilder command.
+
+Example
+```sh
+./hack/kubebuilder.sh create webhook --group anywhere --version v1alpha1 --programmatic-validation --kind WhateverKind
```

Comment:
This is for validating webhooks. You might want to add another comment for the mutation webhooks with the `--defaulting` flag.